packages/ui/components/validate-messages/translations/de.js

```diff
@@ -5,13 +5,13 @@ export default {
     MinLength: 'Du musst mindestens {params} Zeichen eingeben.',
     MaxLength: 'Du kannst maximal {params} Zeichen eingeben.',
     MinMaxLength: 'Du musst zwischen {params.min} und {params.max} Zeichen eingeben.',
-    Pattern: 'Geben Sie ein gültiges {fieldName} ein.',
-    IsNumber: 'Geben Sie ein gültiges {fieldName} ein.',
+    Pattern: 'Geben Sie ein gültige {fieldName} ein.',
+    IsNumber: 'Geben Sie ein gültige {fieldName} ein.',
```

Comment:
What kind of fieldName is expected here  for "IsNumber"?

Reply:
Fieldnames can be anything, e.g. amount, interest, number of guests.

Reply:
Coming up with generic German messages when we have a variable fieldName, seems to be impossible. Also see this discussion: https://github.com/ing-bank/lion/pull/1938#issuecomment-1468390611 
A good and easy way to mitigate this (for >99% of cases) would be https://github.com/ing-bank/lion/issues/1940